Doc/library/doctest.rst

```diff
@@ -248,16 +241,52 @@ Like :func:`testmod`, :func:`testfile`'s verbosity can be set with the
 ``-v`` command-line switch or with the optional keyword argument
 *verbose*.
 
-There is also a command line shortcut for running :func:`testfile`.  You can
-instruct the Python interpreter to run the doctest module directly from the
-standard library and pass the file name(s) on the command line::
+There is also a command line shortcut for running :func:`testfile`, see section
+:ref:`doctest-cli`.
 
-   python -m doctest -v example.txt
+For more information on :func:`testfile`, see section :ref:`doctest-basic-api`.
 
-Because the file name does not end with :file:`.py`, :mod:`doctest` infers that
-it must be run with :func:`testfile`, not :func:`testmod`.
 
-For more information on :func:`testfile`, see section :ref:`doctest-basic-api`.
+.. _doctest-cli:
+
+Command-line Usage
+------------------
+
+The :mod:`doctest` module can be invoked as a script from the command line:
+
+.. code-block:: bash
+
+   python -m doctest [-v] [-o OPTION] [-f] file [file ...]
+
+.. program:: doctest
+
+.. option:: -v, --verbose
+
+   Detailed report of all examples tried is printed to standard output,
+   along with assorted summaries at the end::
+
+      python -m doctest -v example.py
+
+   This will import :file:`example.py` as a standalone module and run
+   :func:`testmod` on it.  Note that this may not work correctly if the file is
+   part of a package and imports other submodules from that package.
+   If the file name does not end with :file:`.py`, :mod:`doctest` infers that
+   it must be run with :func:`testfile`, not :func:`testmod`::
+
+      python -m doctest -v example.txt
```

Comment:
```suggestion
   This will import :file:`example.py` as a standalone module and run
   :func:`testmod` on it. Note that this may not work correctly if the
   file is part of a package and imports other submodules from that package.

   If the file name does not end with :file:`.py`, :mod:`!doctest` infers
   that it must be run with :func:`testfile` instead::

      python -m doctest -v example.txt
```